core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -1110,6 +1111,61 @@ class SubmissionApiServiceSpec extends ApiServiceSpec with TableDrivenPropertyCh
     }
   }
 
+  it should "return 400 Bad Request if the outputPath is an external bucket" in {
```

Comment:
Okay. The worry that I had in particular was possibly being able to save outputs from an auth-domain protected workspace to a different bucket with different permissions. There's not much we can do to prevent people from going outside of Terra and explicitly copying data out of one bucket and into another, but any features we build in should respect the access control rules that we have in place.

Reply:
For sure- we could disable external bucket outputs in the case of AD'd workspaces, if we ever get there